### PR TITLE
Add Peeked Markers to Map

### PR DESCRIPTION
**Pull request by BurningSkyMK1**
This adds the ability to track what player(s) have peeked at an unrevealed public objective, either stage 1 or 2, and draw markers onto the map to display this information.

The goal of this feature is to better convey to the table which public objectives player(s) have used the Espionage SC on.

### CODE REVIEW

**Comment by bradleysigma** [comment]
I would suggest changing each instance of "peak" (mountain top) to "peek" (glance).

**Comment by BurningSkyMK1** [comment]
Done @bradleysigma. I only changed the code I directly worked with from 'peaked' to 'peeked'. I.e., I didn't update `publicObjectives1Peakable` or `publicObjectives2Peakable`, as that touches code outside the scope of my changes.

File: src/main/java/ti4/AsyncTI4DiscordBot.java [hunk 1]
```diff
@@ -375,6 +377,7 @@ private static void initializeWhitelistedRoles() {
         bothelperRoles.add(jda.getRoleById("1063464689218105354")); // FoW Server Game Admin
         bothelperRoles.add(jda.getRoleById("1131925041219653714")); //Jonjo's Server
         bothelperRoles.add(jda.getRoleById("1215450829096624129")); //Sigma's Server
+        bothelperRoles.add(jda.getRoleById("1225597399385374781")); //Sigma's Server
```
Comment (jamesrudolph) [inline src/main/java/ti4/AsyncTI4DiscordBot.java:380]: Forgot to update comment

Comment (BurningSkyMK1) [reply src/main/java/ti4/AsyncTI4DiscordBot.java:380]: Oops, good catch @jamesrudolph.

Comment (BurningSkyMK1) [reply src/main/java/ti4/AsyncTI4DiscordBot.java]: Fixed.

**Pull request merged**